app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,59 @@
+class AlbumsController < ApplicationController
+
+  def index
+    @albums = Album.all
+  end
+
+  def upvote
+    @album = Album.find(params[:id])
+    @album.update_attribute(:ranking, @album.ranking + 1)
+    redirect_to action: :show
```

Comment:
We should use a route helper method for this: `redirect_to album_path(@album.id)` as we do in other places.